.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    # Files stored in repository root
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
We might want to check the changes only weekly or monthly so that Dependabot does not generate too much noise.

Reply:
Good point.  Monthly sounds like a good tradeoff.

Reply:
Please ignore my previous comment.  I though I was commenting on a https://github.com/csutils/cscppc pull request.